app/src/main/java/com/sdpteam/connectout/profile/editProfile/EditProfile.java

```diff
@@ -0,0 +1,36 @@
+package com.sdpteam.connectout.profile.editProfile;
+
+import android.net.Uri;
+
+import com.sdpteam.connectout.profile.Profile;
+import com.sdpteam.connectout.profile.ProfileDataSource;
+import com.sdpteam.connectout.remoteStorage.FileStorageFirebase;
+
+import java.util.concurrent.CompletableFuture;
+
```

Comment:
Please add a few lines of documentation. Its always better to be able to understand some code quickly